packages/manager/modules/anthos/src/dashboard/routing.js

```diff
@@ -112,7 +112,13 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.href('anthos.dashboard.access-restriction', { serviceName }),
 
       serviceInfo: /* @ngInject */ (serviceName, AnthosTenantsService) =>
-        AnthosTenantsService.getServiceInfo(serviceName),
+        AnthosTenantsService.getServiceInfo(serviceName).then((data) => {
+          const serviceInfo = {
+            ...data,
+            serviceType: SERVICE_TYPE,
+          };
+          return serviceInfo;
+        }),
```

Comment:
Just a suggestion (not a blocker)
```suggestion
        AnthosTenantsService.getServiceInfo(serviceName).then((data) => ({
            ...data,
            serviceType: SERVICE_TYPE,
        }),
````